qiskit_ibm_runtime/qpy/exceptions.py

```diff
@@ -12,7 +12,7 @@
 
 """Exception for errors raised by the pulse module."""
 from typing import Any
-from qiskit.qpy.exceptions import QpyError
+from qiskit.qpy.exceptions import QpyError, QiskitWarning
```

Comment:
```suggestion
from qiskit.exceptions import QiskitWarning
from qiskit.qpy.exceptions import QpyError
```

Reply:
Thanks for brining this up, i'll cherry pick those PRs